Makefile

```diff
@@ -1,7 +1,8 @@
 SHELL := '/bin/bash'
 
 test:
-	python form_models/tests/run.py
+	coverage run form_models/tests/run.py
+	coverage report
```

Comment:
And the option that ignores 100% coverage lines?  I can't remember what it's called. :)